guacamole/src/main/webapp/translations/en.json

```diff
@@ -475,6 +476,14 @@
         "FIELD_OPTION_FONT_SIZE_96"    : "96",
         "FIELD_OPTION_FONT_SIZE_EMPTY" : "",
 
+        "FIELD_OPTION_TERMINAL_TYPE_EMPTY" : "",
+        "FIELD_OPTION_TERMINAL_TYPE_XTERM" : "xterm",
+        "FIELD_OPTION_TERMINAL_TYPE_XTERM_256COLOR" : "xterm-256color",
+        "FIELD_OPTION_TERMINAL_TYPE_VT220" : "vt220",
+        "FIELD_OPTION_TERMINAL_TYPE_VT100" : "vt100",
+        "FIELD_OPTION_TERMINAL_TYPE_ANSI"  : "ansi",
+        "FIELD_OPTION_TERMINAL_TYPE_LINUX" : "linux",
```

Comment:
This is a style nitpick, but in keeping with the style used throughout the rest of these JSON files, if there are a group of objects like this, the ":" should be lined up in the same column.